Tools/msi/uploadrelease.proj

```diff
@@ -8,6 +8,7 @@
         <DownloadUrlBase Condition="'$(DownloadUrlBase)' == ''">$(TARGET)</DownloadUrlBase>
         <DownloadUrlBase Condition="'$(DownloadUrlBase)' == ''">/srv/www.python.org/ftp/python</DownloadUrlBase>
         <IncludeDoc Condition="'$(IncludeDoc)' == ''">true</IncludeDoc>
+        <BuildForRelease Condition="'$(BuildForRelease)' == ''">true</BuildForRelease>
```

Comment:
This determines whether the version is "3.6.1rc1" (building for release) or "3.6.1dev1234" (not building for release).